src/dotnet-scaffolding/dotnet-scaffold-aspnet/Templates/BlazorIdentity/IdentityComponentsEndpointRouteBuilderExtensions.Interfaces.cs

```diff
@@ -0,0 +1,8 @@
+using Microsoft.DotNet.Scaffolding.TextTemplating;
+
+namespace Microsoft.DotNet.Tools.Scaffold.AspNet.Templates.BlazorIdentity;
+
+public partial class IdentityComponentsEndpointRouteBuilderExtensions : ITextTransformation
```

Comment:
Can you give me the two sentence overview of what this is (and other similar classes below are) for? I see that the other side of the partial declaration comes from the generated code from the .tt files, but I'm not sure I understand the purpose of this side of it.

Reply:
yep I never really explained that mb We defined the `ITextTransformation` class to help with separation of concerns and define a simple interface we can interact with the auto-generated templated code and add on top of it (if needed) without modifying these auto-generated files. The `.Interfaces.cs` files help with the interface declaration without modifying the auto-generated template file. There might be a better way we can come up with eventually but this seems ok enough for now.